### ISSUE TITLE
ip allocated in vcd_vapp shows constant need to change

### ISSUE THREAD

**Issue by gregswift** [terraform-providers/terraform-provider-vcd#29]
### Terraform Version
Terraform v0.9.11 (did not see this issue on 0.9.5)

### Affected Resource(s)
vcd_vapp

### Terraform Configuration Files
```hcl
variable "artifactory_catalog_name" { default = "Images" }
variable "artifactory_template"     { default = "centos-7" }
variable "artifactory_network"      { default = "VLAN664" }
variable "artifactory_count"        { default = 0 }

resource "vcd_vapp" "artifactory" {
    count         = "${var.artifactory_count}"
    name          = "${format("artifactory-%03d", count.index + 1)}.example.net"
    catalog_name  = "${var.artifactory_catalog_name}"
    template_name = "${var.artifactory_template}"
    ip            = "allocated"
    network_name  = "${var.artifactory_network}"

    metadata {
        role    = "artifactory"
        env     = "prod"
        version = "v1"
    }

}
```

### Debug Output
https://gist.github.rackspace.com/greg5320/7aa1cfd9c321cd055a484d3515be47f9

### Panic Output
no crash

### Expected Behavior
When re-running terraform shouldnt consider the ip address a state change.

```0 to add, 0 to change, 0 to destroy```

### Actual Behavior
(i'm shortening to a single host for this output, but it does this for *every* host with `ip => "allocated")

```
vcd_vapp.artifactory: Refreshing state... (ID: artifactory-001.example.net)
...
~ vcd_vapp.artifactory
    ip: "10.10.155.55" => "allocated"
...
0 to add, 1 to change, 0 to destroy
```

### Steps to Reproduce
1. `terraform plan` or `apply`

### Important Factoids
Didn't see this until we upgraded from 0.9.5 to 0.9.11
Here are the version I've used:

terraform-0.8.7
terraform-0.9.1
terraform-0.9.3
terraform-0.9.5
terraform-0.9.11

We have to use `allocated` because of the way our network is setup. See #17

**Comment by gregswift** [comment]
@githubvick confirms in #17 to be seeing the same thing

**Comment by bobbydeveaux** [comment]
@gregswift: fixed in https://github.com/terraform-providers/terraform-provider-vcd/pull/37 - just waiting CI suite tests to pass

**Comment by gregswift** [comment]
confirmed fixed in 1.0.0 of vcd provider
